tools/send-to-caas/send-utils.js

```diff
@@ -249,6 +249,14 @@ const getTags = (s) => {
 const getDateProp = (dateStr, errorMsg) => {
   if (!dateStr) return undefined;
   try {
+    if (dateStr.length === 10) {
+      const [m, d, y] = dateStr.split('-');
+      if (m > 12) {
```

Comment:
Month greater than 12?

Reply:
That means that some dates will be misleading, but I suppose you don't have any way of knowing?

Reply:
Yep.  No way of knowing.  The alternative is to fail and force authors to fix it in the metadata.

Reply:
Got it. Thanks for the explanation.